src/ProjectTemplates/Web.ProjectTemplates/content/Grpc-Application/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "gRPC Application",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1027"
```

Comment:
@phenning Can we reserve new ids for the new grpc templates?

Reply:
You may also want to adjust the order below to make sure that this appears in the proper place in the list relative to the other templates in the dialog.

Reply:
But I've added the two strings here as 1027 and 1028 in the resources. These were the next available IDs.